docs/_theme/section.hbs

```diff
@@ -142,4 +142,11 @@
     <div class='pad1y quiet space-top2 prose-big'>Events</div>
     <%= renderSectionList({ members: section.members.events, renderSection: renderSection, noun: 'Event' }) %>
   <% } %>
+
+  <% if (section.sees) { %>
+    <div class='pad1y quiet space-top2 prose-big'>See</div>
```

Comment:
@lucaswoj I've gone with using "Related" instead of "See" so we can include links to other locations if necessary